build/pxc-entrypoint.sh

```diff
@@ -254,6 +254,12 @@ else
 
 fi
 
+WSREP_CLUSTER_NAME=$(grep wsrep_cluster_name  ${CFG}| cut -d '=' -f 2| tr -d ' ' )
+if [[ !  ${WSREP_CLUSTER_NAME} || ${WSREP_CLUSTER_NAME} == 'noname' ]]; then
```

Comment:
Two spaces between `!` and `${WSREP_CLUSTER_NAME}`

Reply:
done